ci/ansible.cfg

```diff
@@ -1,5 +1,5 @@
 [defaults]
-callbacks_enabled = custom_logger
+callbacks_enabled = custom_logger, junit
```

Comment:
How does this relate to the change here: https://github.com/infrawatch/feature-verification-tests/pull/132
Are they alternatives, or complimentary?

Reply:
its not related, in this PR Munesha is enable the callback plugin only, but in my PR I was propose another solution  trying to customise the plugin.  as I have the feeling that we can't use the default one directly .